### ISSUE TITLE
HTML tag changes break tests

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#8091]
```
org.junit.ComparisonFailure: expected:<"Kinder erfreuen.<br[ /><br /]>Make children happy...> but was:<"Kinder erfreuen.<br[><br]>Make children happy...>
```

As the trackable seems orphaned, I assume either a small website change/glitch.
Or did we have any related changes causing this html tag changing?

**Comment by bekuno** [comment]
there is also a second error:
~~~
testSearchByGeocodeBasis(cgeo.geocaching.CgeoApplicationTest)
java.lang.AssertionError: [description]
Expecting:
<"<img src="http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg">">
to start with:
<"<img src="http://img.geocaching.com/cache/large/1711f8a1-796a-405b-82ba-8685f2e9f024.jpg" />">
~~~

**Comment by Lineflyer** [comment]
seems they upgraded to "real HTML" :)
I will fix this within the same PR.

**Comment by Lineflyer** [comment]
OK, there are more follow up errors with changes HTML tags.
Looks like we need to update mocked data. Unassigned myself for now due to lack of time.

**Comment by Lineflyer** [comment]
If someone wants to tackle this, please base it on `release`

**Comment by bekuno** [comment]
Next one: GC3XX5J
Lets wait until changes are finished.

**Comment by bekuno** [comment]
fixed by #8094
